### PR TITLE
Add audio balance and badge in left desktop panel

### PR DESCRIPTION
**Pull request by sddioulde**
### Description

Display user's audio balance and badge in left nav

Screen Shot 2021-11-10 at 8 46 57 AM

Screen Shot 2021-11-10 at 8 47 59 AM

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?

### How Has This Been Tested?

Locally against:
- account with positive audio balance
- account with zero audio balance

### How will this change be monitored?

For features that are critical or could fail silently please describe the monitoring/alerting being added.

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/sd-audio-panel

File: src/containers/nav/desktop/NavAudio.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  const account = useSelector(getAccountUser)
+  const totalBalance = useSelector(getAccountTotalBalance) ?? null
+  const nonNullTotalBalance = totalBalance !== null
+  const positiveTotalBalance =
+    nonNullTotalBalance && totalBalance!.gt(new BN(0))
+  // we only show the audio balance and respective badge when there is an account
+  // so below null-coalescing is okay
+  const { tier } = useSelectTierInfo(account?.user_id ?? 0)
+  const audioBadge = audioTierMapPng[tier]
+  console.log({ totalBalance, tier })
```
Comment (jowlee) [inline src/containers/nav/desktop/NavAudio.tsx:30]: log

File: src/containers/nav/desktop/NavAudio.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        {formatWei(totalBalance!, true, 0)}
+      </span>
+    </div>
+  ) : nonNullTotalBalance ? (
+    <div className={styles.audio}>
+      <img alt='no tier' src={IconNoTierBadge} width='16' height='16' />
+      <span className={styles.audioAmount}>
+        {formatWei(totalBalance!, true, 0)}
+      </span>
+      <span className={styles.earnAudio}>
```
Comment (sddioulde) [inline src/containers/nav/desktop/NavAudio.tsx:49]: Is this supposed to link to somewhere? the /audio page maybe? @raymondjacobson

File: src/containers/nav/desktop/NavAudio.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        <span>{messages.earnAudio}</span>
+        <IconCaretRight className={styles.earnAudioCaret} />
+      </span>
+    </div>
+  ) : (
+    <div className={styles.audio} />
+  )
```
Comment (jowlee) [inline src/containers/nav/desktop/NavAudio.tsx:56]: what is the third case? Isn't it if the user has a balance or not?

File: src/containers/nav/desktop/NavColumn.js [hunk 4]
```diff
@@ -223,29 +225,32 @@ const NavColumn = ({
...
+                  </div>
+                  <div className={styles.handleContainer}>
+                    <span
+                      className={styles.handle}
+                      onClick={goToProfile}
+                    >{`@${handle}`}</span>
+                  </div>
                 </div>
               </div>
+              <NavAudio />
```
Comment (jowlee) [inline src/containers/nav/desktop/NavColumn.js:253]: Should showing this be wrapped in a feature flag to turn on w. rewards? I'm not sure

**Review by jowlee — APPROVED** [review]

File: src/containers/nav/desktop/NavAudio.tsx (hunk 3 above)
Comment (sddioulde) [reply src/containers/nav/desktop/NavAudio.tsx:59]: It's just for the styling. to keep the empty space there

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/sd-audio-panel

File: src/containers/nav/desktop/NavAudio.tsx (hunk 2 above)
Comment (raymondjacobson) [reply src/containers/nav/desktop/NavAudio.tsx:53]: yep!

File: src/containers/nav/desktop/NavColumn.js (hunk 4 above)
Comment (raymondjacobson) [reply src/containers/nav/desktop/NavColumn.js:252]: yep! let's feature flag. a new one would be preferable

**Review by raymondjacobson — APPROVED** [review]

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/sd-audio-panel

**Pull request merged**